crates/stackable-operator/src/commons/cache.rs

```diff
@@ -0,0 +1,86 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::time::Duration;
+
+/// [`TtlCache`] with sensible defaults for a user information cache
+pub type UserInformationCache = TtlCache<30, 10_000>;
+
+/// Least Recently Used (LRU) cache with per-entry time-to-live (TTL) value.
```

Comment:
This could be clearer on that this is *configuring* a cache in the product, not providing a cache to operators.

Reply:
8281fca02d93b1622e0e3b4cfd142d491150e425